### DISCUSSION TITLE
yolov8_node 실행 시 속도가 느립니다.

### DISCUSSION THREAD

**Discussion by imerickkang** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#125] [2024년도 교육 자료 관련 질문]
![image](https://github.com/user-attachments/assets/048f750f-e94b-459a-a973-2aea065df4b1)
코드 실행시 image_publisher_node의 실행속도(0.03)보다 느리게 작동하여 강의영상처럼 부드럽게 동작하지않습니다. 1초 단위로 뚝뚝 끊깁니다. image_publisher_node의 속도와 맞출 수 있는 방법이 있을까요?

**Comment by HyeongKeunHong** [comment]
하드웨어 성능 문제로 보입니다. 일단 그대로 과제를 진행하시면 됩니다. 오프라인 현장에서는 대여용 PC가 지급되므로, 오프라인 현장에서는 해당 문제가 발생하지 않을 것입니다.
